community/kernel/src/main/java/org/neo4j/kernel/impl/storemigration/StoreMigrationTool.java

```diff
@@ -1,130 +0,0 @@
-/*
- * Copyright (c) 2002-2015 "Neo Technology,"
- * Network Engine for Objects in Lund AB [http://neotechnology.com]
- *
- * This file is part of Neo4j.
- *
- * Neo4j is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-package org.neo4j.kernel.impl.storemigration;
-
-import java.io.File;
-import java.io.IOException;
-
-import org.neo4j.io.fs.FileSystemAbstraction;
-import org.neo4j.io.pagecache.PageCache;
-import org.neo4j.kernel.DefaultFileSystemAbstraction;
-import org.neo4j.kernel.GraphDatabaseDependencies;
-import org.neo4j.kernel.api.index.SchemaIndexProvider;
-import org.neo4j.kernel.configuration.Config;
-import org.neo4j.kernel.extension.KernelExtensions;
-import org.neo4j.kernel.impl.logging.LogService;
-import org.neo4j.kernel.impl.logging.StoreLogService;
-import org.neo4j.kernel.impl.spi.KernelContext;
-import org.neo4j.kernel.impl.storemigration.legacystore.LegacyStoreVersionCheck;
-import org.neo4j.kernel.impl.storemigration.monitoring.VisibleMigrationProgressMonitor;
-import org.neo4j.kernel.impl.util.Dependencies;
-import org.neo4j.kernel.lifecycle.LifeSupport;
-import org.neo4j.logging.FormattedLogProvider;
-import org.neo4j.logging.Log;
-import org.neo4j.logging.LogProvider;
-
-import static java.lang.String.format;
-import static org.neo4j.kernel.extension.UnsatisfiedDependencyStrategies.ignore;
-import static org.neo4j.kernel.impl.pagecache.StandalonePageCacheFactory.createPageCache;
-import static org.neo4j.kernel.impl.storemigration.StoreUpgrader.NO_MONITOR;
-
-/**
- * Stand alone tool for migrating/upgrading a neo4j database from one version to the next.
- *
- * @see StoreMigrator
- */
-public class StoreMigrationTool
```

Comment:
Could this have been a tool used out in the wild and we are supposed to have and now we just deleted?